echopype/core.py

```diff
@@ -45,7 +45,7 @@ def inner(test_ext: str):
         "parser": ParseAZFP,
         "set_groups": SetGroupsAZFP,
         "concat_dims": {
-            "platform": None,
+            "platform": "time2",
             "nmea": "time1",
```

Comment:
At the moment I am not very confident in how xarray's `combine_nested` works. I think right now line 200 actually makes it so that none of these parameters are actually used. This seems like a larger issue that needs to be investigated. I suggest that we open up a new issue and deal with this after 0.6.0, @emiliom if you agree, can you open up an issue for this too?

Reply:
What would be the scope of the issue? If it's beyond my narrow comment on the `nmea` group, I think it's best if you open the issue instead. Otherwise I can open it myself.

Reply:
I would say that we need an issue to investigate the use of `concat_dims` in `combine_nested`. From my comment above, it looks like only the `default` option is ever used.